.github/workflows/build-test.yaml

```diff
@@ -16,17 +16,14 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - uses: actions/checkout@v3
-    - name: Install dependencies
+    - name: Install osmesa
       run: |
         sudo apt-get update && sudo apt-get upgrade
         sudo apt install libgl1-mesa-dri libegl1-mesa libgbm1 libosmesa6 libosmesa6-dev
```

Comment:
i'm not sure all these are needed... the osmesa install instructions just say `sudo apt-get install libosmesa6-dev`. So I think I'll try editing this line after tests pass in this PR.